cli/pack/common.go

```diff
@@ -28,14 +28,24 @@ var (
 			`^(?P<Major>\d+)\.(?P<Minor>\d+)\.(?P<Patch>\d+)-(?P<Count>\d+)-(?P<Hash>g\w+)$`,
 		),
 	}
+
+	// Check if --version or --suffix is allowed to be added in package version.
+	packageVerAllowed = map[string]*regexp.Regexp{
+		// The version number is normally taken verbatim from the package's version.
+		// The only restriction placed on the version is that it cannot contain a dash "-".
+		// http://ftp.rpm.org/max-rpm/ch-rpm-file-format.html
+		RpmType: regexp.MustCompile(`^[^-]+$`),
+		// upstream-version must satisfy `[-a-zA-Z0-9.+:]+`.
+		// https://www.debian.org/doc/manuals/debian-reference/ch02.en.html#_debian_package_file_names
+		DebType: regexp.MustCompile(`^[-a-zA-Z0-9.+:]+$`),
```

Comment:
A tilde possibly should be here as well. See https://github.com/knazarov/mkrepo/blob/cdc6ef352476c2ed7ec18cb38583328deaf1baf5/debrepo.py#L450-L460.

Reply:
Thanks, added